airflow-core/src/airflow/api_fastapi/core_api/openapi/exceptions.py

```diff
@@ -26,7 +29,30 @@ class HTTPExceptionResponse(BaseModel):
     detail: str | dict
 
 
-def create_openapi_http_exception_doc(responses_status_code: list[int]) -> dict:
+class HTTPExceptionValue(TypedDict):
+    """Structure for the value field in an HTTP exception example."""
+
+    detail: str | dict
+
+
+class HTTPExceptionExample(TypedDict, total=False):
```

Comment:
Let's ensure that there will be all fields for `HTTPExceptionExample`

```suggestion
class HTTPExceptionExample(TypedDict, total=True):
```